### ISSUE TITLE
[GUI] Balance and payments not matching in home (payments display w/ delay)

### ISSUE THREAD

**Issue by pythcoiner** [wizardsardine/liana#980]
Doing some test w/ a regtest that have a halving at 1000 blocks (instead 150), some received coin missing on home page:

![image](https://github.com/wizardsardine/liana/assets/124568858/aa79d0cd-0cca-4170-885b-9b81fb3930f7)

but appear in coins page:

![image](https://github.com/wizardsardine/liana/assets/124568858/efc023d2-b316-4d38-be0c-abce7c19ed12)

edit: the coins appear in home payments after few minutes but where present in coins from start...

(i'm doing address reuse in this context)

to reproduce:
- generate 1 coin to Liana
- genarate 1 coin to another wallet
- generate 1 coin to Liana
- genarate 999 coins to another wallet
- generate 1 coin to Liana
- genarate 999 coins to another wallet
- etc...

**Comment by pythcoiner** [comment]
continuing generate coins:

![image](https://github.com/wizardsardine/liana/assets/124568858/67b48595-94c9-4b29-a057-e1c05964adc2)

![image](https://github.com/wizardsardine/liana/assets/124568858/0d12c67a-b6b0-4058-8f6d-62bb544faf6f)

and coins are still not yet displayed after minutes on home received payments

**Comment by pythcoiner** [comment]
then some coins are missing on home:

![image](https://github.com/wizardsardine/liana/assets/124568858/d19beee7-a007-4ca1-ae87-a26732892ac7)

but all present in coins:

![image](https://github.com/wizardsardine/liana/assets/124568858/130167d7-b3ed-4784-9e7d-c0c4dfa73301)

**Comment by darosior** [comment]
Good catch. I think the GUI is also querying the transaction which created those coins in order to estimate the fee. And IIRC it would not display the coin if it couldn't. cc @edouardparis am i right? Do you think it could be due to something else?
